src/main/resources/application.yaml

```diff
@@ -184,7 +184,7 @@ os-postcode-lookup:
 
 stitching:
   api:
-    url: "${EM_CCD_ORCHESTRATOR_URL:http://localhost:8084}/api/stitch-ccd-bundles"
+    url: "${EM_CCD_ORCHESTRATOR_URL:http://em-ccd-orchestrator-aat.service.core-compute-aat.internal}/api/stitch-ccd-bundles"
```

Comment:
@kurt-uk not sure these changes are required anymore - This was to get it working on preview (pointing to AAT URL)